barretenberg/cpp/src/barretenberg/sumcheck/sumcheck.hpp

```diff
@@ -711,13 +707,16 @@ template <typename Flavor, size_t virtual_log_n = CONST_PROOF_SIZE_LOG_N> class
 
         // For ZK Flavors: the evaluation of the Row Disabling Polynomial at the sumcheck challenge
```

Comment:
This comment seems to ignore the libra component - can you give it a quick update?